android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -639,6 +622,11 @@ public void showErrorDialog(String errorType, String errorDesc) {
         builder.show();
     }
 
+    public void showErrorSnackbar(String errorType, String errorDesc) {
```

Comment:
Snackbar will not able to show whole error message in one line. That's why i'm using it.